css/vendors.css

```diff
@@ -2895,6 +2895,13 @@ th {
   text-align: left
 }
 
+.edit {
+  padding-left: .25rem;
+  vertical-align: top;
+  font-size: .75rem;
+  color: rgba(0, 0, 0, .54)
+}
```

Comment:
Let's move this to `css/devcenter.css`. The vendors file has a bunch of third party styles and my fear is this file may be removed someday as we remove dependencies.

Reply:
Thx! Can I leave that to you?

Reply:
I'd prefer for the developer who opens the PR to own all code changes. I've run into issues in the past where two devs both work on a branch and something gets messed up because of a miscommunication between the two.

Reply:
Got it. Will do!